docs/technical/preservation/implementation_diagram.md

```diff
@@ -0,0 +1,175 @@
+    dashboard.
+
+### Fixity status show page display
+Scenario: Looking at a Scanned Resource
+
+We display fixity summary in three places:
+
+#### Resource Show Page
+
+https://github.com/pulibrary/figgy/blob/c106ea719f9473e0fc3d0bfa608da0d345ab4a94/app/views/catalog/_resource_attributes_default.html.erb#L30-L39
+
+
+It uses a decorator which uses a helper. The helper makes queries about failed
+  and succeeded fixity checks.
+  * https://github.com/pulibrary/figgy/blob/c106ea719f9473e0fc3d0bfa608da0d345ab4a94/app/decorators/valkyrie/resource_decorator.rb#L89-L99
+  * https://github.com/pulibrary/figgy/blob/c106ea719f9473e0fc3d0bfa608da0d345ab4a94/app/helpers/fixity_dashboard_helper.rb#L40-L42
+
+#### Member Resource List on Show Page (for MVW)
+
+There's also fixity_badges, which is used in the member_resources list. Look at this next time.
+
+We call fixity_badges from here:
+
+https://github.com/pulibrary/figgy/blob/030a01df143cebe3c3c8a8a09d7ccb4be1170a7f/app/views/catalog/_member_resources.html.erb#L50
+
+This displays the local fixity count, but with no status. The status helper
+calls the count method.
+
+#### File Set Show Page
+
+https://github.com/pulibrary/figgy/blob/030a01df143cebe3c3c8a8a09d7ccb4be1170a7f/app/views/catalog/_members_file_set.html.erb
+
+This has a column for local fixity and cloud fixity for every FileMetadata, but
+only the primary file will ever have a populated success/failure.
+
+https://github.com/pulibrary/figgy/blob/030a01df143cebe3c3c8a8a09d7ccb4be1170a7f/app/views/catalog/_file_detail.html.erb
+
+This uses the same helpers, `format_fixity_success` and
+`format_cloud_fixity_success`, but also displays the last success date for each.
+
+### Fixity Dashboard
+
+Note that it's slow. Takes a minute to load due to
+https://github.com/pulibrary/figgy/issues/5545
+
+The code here is very straightforward. See FixityDashboardController and
+accompanying template / partials.
+
+### Creation of Tombstones
+
+There's a CreateTombstone before delete hook in the change set persister.
+
+It saves some identifiers and embeds the preservation object.
+
+The Tombstone model: https://github.com/pulibrary/figgy/blob/main/app/models/tombstone.rb
+
+### Restore Tombstones
+
+When a fileset has been deleted there's a "Deleted Files" section in the File
+Manager. Each deleted file is listed by title with a "Reinstate" button.
+
+There's a `child_tombstones` method on the wayfinder to power this list. Uses
+the parent_id stored on the tombstone. Currently only implemented for
+ScannedResources.
+
+Uses tombstone_restore_ids to pass the value to the restore_tombstones.rb change_set_persister callback.
+- The importer pulls the metadata and binary down from the cloud.
+- The metadata is converted back into a FileSet object using the Valkyrie Sequel ORM converter. This restores id, created date, updated date, and internal resource.
+- Currently intermediate files are not preserved/restorable.
```

Comment:
This seems resolved -- fully? 
preservation
https://github.com/pulibrary/figgy/blob/dcfe59e82e8723793424084c0ee78bf323251cb9/app/services/preserver.rb#L73-L75
restoration
https://github.com/pulibrary/figgy/blob/dcfe59e82e8723793424084c0ee78bf323251cb9/app/services/preserver/importer.rb#L15

The deletion marker stores the preservation object so it has the info for all the relevant binary files.